src/app/rundown-view/components/rundown-tooltips/piece-tooltip/piece-tooltip.component.scss

```diff
@@ -27,6 +27,10 @@
     sofie-icon-button {
       color: var(--black-color);
     }
+
+    span {
+      margin-left: 4px;
+    }
```

Comment:
Instead of setting margins in CSS we should start to utilize our classes for this. So we should add `ms-1` on the span element instead of writing a class for it.

Reply:
I have updated it to use `ms-1`.